external/java/pom.xml

```diff
@@ -6,7 +6,29 @@
 
     <groupId>ch.epfl.dedis</groupId>
     <artifactId>cothority</artifactId>
-    <version>v2.0.0-dev</version>
+    <version>2.0.1</version>
+    <name>Cothority client library for Java</name>
+    <description>A client library for communicating with cothorities (a collective authority).</description>
+    <url>https://github.com/dedis/cothority</url>
+    <licenses>
+        <license>
+            <name>GNU Affero General Public License version 3</name>
+            <url>https://opensource.org/licenses/AGPL-3.0</url>
+        </license>
+    </licenses>
+    <developers>
+        <developer>
+            <name>DEDIS Sofware Engineers</name>
+            <email>cothority@googlegroups.com</email>
+            <organization>DEDIS Lab, EPFL</organization>
+            <organizationUrl>https://dedis.epfl.ch/</organizationUrl>
+        </developer>
+    </developers>
+    <scm>
+        <connection>scm:git:git://github.com/dedis/cothority.git</connection>
+        <developerConnection>scm:git:ssh://github.com:dedis/cothority.git</developerConnection>
+        <url>http://github.com/dedis/cothority/tree/master</url>
```

Comment:
Does the `<scm>` tag give us some kind of functionality? If it does, then it might not work for people who want to publish their own versions (see the other comment).

Reply:
That's all copy-pasta from Sonatype. I have no idea what it's doing.